grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ExplanationTest.java

```diff
@@ -0,0 +1,121 @@
+package ai.grakn.test.graql.reasoner;
```

Comment:
I think this is missing the copy right info. @aelred I though checkstyle looked for this? Does it skip the check for tests?